### PR TITLE
Translate Yoshida_1.md

### LINKED ISSUE #91: Yoshida_1 英訳
原文
https://github.com/scalajp/2017.scalamatsuri.org/blob/gh-pages/_candidates_ja/Yoshida_1.md

### PR DESCRIPTION
**Pull request by okapies**
Fix #91
/review @smroy

### CODE REVIEW

File: _candidates_en/Yoshida_1.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+audience: Intermediate
+language: Japanese
+twitter: grimrose
+github: grimrose
+organization: Yokohama.groovy
+tags:
+  - Tools
+  - Others
+---
+ScalikeJDBC is a powerful and flexible DB access library. In combination with Skinny ORM, we can create a better relationship with the database. I'll introduce a case-study utilizing SQL to tackle the difficulty of database and friendly CRUD API based on one year work experience.
```
Comment (smroy) [inline _candidates_en/Yoshida_1.md]: Line 1: Should "DB" be written out as "data base"?
Line 2: What is "database" - does this need to be more specific?
Line 2: Again, should "database" be specific?
Line 2: "friendly" does not make sense in this context, but I don't know the subject area well enough to make an inference as to how I should edit this.
Line 3: Is "one year work experience" referring to one year running this program?

**Review by smroy — CHANGES_REQUESTED** [review]
Line 1: Should "DB" be written out as "data base"?
Line 2: What is "database" - does this need to be more specific?
Line 2: Again, should "database" be specific?
Line 2: "friendly" does not make sense in this context, but I don't know the subject area well enough to make an inference as to how I should edit this.
Line 3: Is "one year work experience" referring to one year running this program?

File: _candidates_en/Yoshida_1.md (hunk 1 above)
Comment (eed3si9n) [reply _candidates_en/Yoshida_1.md]: > Line 1: Should "DB" be written out as "data base"?

Yes, but as one word _database_. In general though we can assume that the audience knows what a database is and does. I think the question is _a_ vs _the_. It's not like _the_ fridge where we can assume that everyone has one in their kitchen. So it's more like _a_ database in my opinion.

> Line 2: What is "database" - does this need to be more specific?

How about: In conjunction with Skinny ORM, it provides better handle to dealing with databases.

> Line 2: "friendly" does not make sense in this context, but I don't know the subject area well enough to make an inference as to how I should edit this.
> 
> Line 3: Is "one year work experience" referring to one year running this program?

How about: I'll share my experience of using them in production for a year: SQL provides more direct access to the database, whereas the CRUD API provides easier to use wrapper around the database.

**Review by smroy — COMMENTED** [review]
I've made a few edits based on your comments; if this looks ok to you I will go ahead and approve. Thanks.

**Review by eed3si9n — APPROVED** [review]

**Comment by eed3si9n** [comment]
Yea, I think it looks close enough to the Japanese original, which was written in somewhat vague way.

**Comment by grimrose** [comment]
thanks 👍

**Comment by okapies** [comment]
@smroy: Thanks for reviewing!

**Comment by smroy** [comment]
@okapies Of course! I hope it was helpful.

**Pull request merged**